boards/seeedstudio-gd32/include/periph_conf.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright (C) 2020 Koen Zandberg <koen@bergzand.net>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_seeedstudio-gd32 SeeedStudio GD32 RISC-V board
+ * @ingroup     boards
+ * @brief       Support for the SeeedStudio GD32 RISC-V board
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the SeeedStudio GD32 RISC-V board
+ *
+ * @author      Koen Zandberg <koen@bergzand.net>
+ */
+
+#ifndef PERIPH_CONF_H
+#define PERIPH_CONF_H
+
+#include "macros/units.h"
+#include "periph_cpu.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define CLOCK_CORECLOCK     MHZ(104)           /**< CPU clock frequency in Hz */
+#define CLOCK_AHB           CLOCK_CORECLOCK    /**< Frequency of the AHB bus in Hz */
+#define CLOCK_APB1          CLOCK_CORECLOCK/2  /**< Frequency of the APB1 bus in Hz */
+#define CLOCK_APB2          CLOCK_CORECLOCK    /**< Frequency of the APB2 bus in Hz */
+
+extern void isr_timer2(unsigned irq);
+extern void isr_timer3(unsigned irq);
```

Comment:
This doesn't really look like the best place to have them. The don't look board specific to me.

Reply:
You're right. I'll rework this to something similar to the stm32 uart peripheral